includes/class-wc-tracker.php

```diff
@@ -540,6 +541,47 @@ private static function get_admin_user_agents() {
 		return array_filter( (array) get_option( 'woocommerce_tracker_ua', array() ) );
 	}
 
+	/**
+	 * Get order counts by payment method
+	 *
+	 * @return array
+	 */
+	public static function get_order_counts_by_payment_method() {
+		global $wpdb;
+
+		$orders_by_payment_method = $wpdb->get_results(
+			"
+			SELECT
+			    order_gateway.payment_method AS payment_method,
+			    COUNT( id ) AS orders_count
+			FROM (
+				SELECT
+					orders.ID AS id,
+				    MAX(
+				        CASE
+				            WHEN meta_key = '_payment_method'
+				            THEN meta_value
+				        END
+				    ) AS payment_method
+				FROM {$wpdb->prefix}posts AS orders
+				LEFT JOIN {$wpdb->prefix}postmeta AS order_meta ON order_meta.post_id = orders.ID
+				WHERE orders.post_status in ( 'wc-completed', 'wc-processing', 'wc-refunded' )
```

Comment:
I think we should add `orders.post_type = 'shop_order'` to the WHERE clause, so that we only get one `wp_post` record per refunded order. Also, that lets the query leverage the `type_status_date` key.

Reply:
Thanks @deltaWhiskey 
I've added a commit for that.